scripts/templates/field_details.j2

```diff
@@ -34,13 +34,22 @@ beta::[ {{ fieldset['beta'] }}]
 {# `Description` column -#}
 {#- Beta fields will add the `beta` label -#}
 {% if field['beta'] -%}
-| beta:[ {{ field['beta'] }} ]
+a| beta:[ {{ field['beta'] }} ]
 
 {{ field['description']|replace("\n", "\n\n") }}
 {%- else -%}
-| {{ field['description']|replace("\n", "\n\n") }}
+a| {{ field['description']|replace("\n", "\n\n") }}
```

Comment:
I covered it at the bottom of the PR description:

> Asciidoc supports [additional formatting options](https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/#cell-styles-and-their-operators) in a table cell using the a| operator. This change opens up a better formatting experience for ECS field descriptions.

The main benefit I wanted was for asterisks in unordered lists to render as dots `ᐧ` instead of `*`. The only unexpected formatting side-effect was the `(C)` rendering as © that I fixed here: https://github.com/elastic/ecs/pull/1952/commits/022201f47c02a403f3fa62b77a5e516905920e8c

Reply:
Got it, sorry, missed that